src/cellrank/kernels/_base_kernel.py

```diff
@@ -463,6 +464,88 @@ def _reuse_cache(self, expected_params: Dict[str, Any], *, time: Optional[Any] =
             self._params = expected_params
         # fmt: on
 
+    def _get_mask(self, series, subset):
```

Comment:
Could be a nice little helper function to have; we could also use it in #1166. What do you think, @michalk8?